packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlert.jsx

```diff
@@ -118,16 +122,27 @@ export const LaunchTriggerAlert = ({
         action={
           isKonnectorRunnable && (
             <>
-              {!isInMaintenance && !isKonnectorDisconnected && (
-                <Button
-                  variant="text"
-                  color={isInError ? 'error' : undefined}
-                  size="small"
-                  disabled={running}
-                  label={t('card.launchTrigger.button.label')}
-                  onClick={SyncButtonAction}
-                />
-              )}
+              {!isInMaintenance &&
+                !isKonnectorDisconnected &&
+                (konnectorPolicy.isBIWebView &&
+                isInError &&
+                error.isSolvableViaReconnect() ? (
+                  <OpenOAuthWindowButton
+                    flow={flow}
+                    account={account}
+                    intentsApi={intentsApi}
+                    konnector={konnector}
+                  />
+                ) : (
+                  <Button
+                    variant="text"
+                    color={isInError ? 'error' : undefined}
+                    size="small"
+                    disabled={running}
+                    label={t('card.launchTrigger.button.label')}
+                    onClick={SyncButtonAction}
+                  />
+                ))}
```

Comment:
It seems that `isKonnectorRunnable` is also used here https://github.com/cozy/cozy-libs/blob/b379309603f23d33f7f38c983f9a5ead77f4a516/packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlertMenu.jsx#L28

should we make the same change?

Reply:
I missed that we could also run the konnector in the menu. I will add the same change here

Reply:
Now the synchronize button in menu triggers the same action. Since the ActionMenuItem does not allow an item to be disabled, we check that the extraParams are available before showing the popup or webview